core/src/main/resources/error/error-classes.json

```
@@ -197,7 +197,7 @@
           "If you have to cast <srcType> to <targetType>, you can set <config> as <configVal>."
         ]
       },
-      "CAST_WITH_FUN_SUGGESTION" : {
+      "CAST_WITH_SUGGESTION" : {
```

If you don't like fun, lets just fix the word: fun -> func
```suggestion
      "CAST_WITH_FUNC_SUGGESTION" : {
```
Don't think we should make the name more generic since there is another error class with a suggestion of a config, see above `CAST_WITH_CONF_SUGGESTION`.